specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-07-01/examples/availabilitySetExamples/AvailabilitySet_Create_WithScheduledEventProfile.json

```diff
@@ -0,0 +1,45 @@
+{
+  "parameters": {
```

Comment:
this example is not referenced anywhere, causing avocado CI check failure. 
You can add this example under the availabilitySet Put call